js/packages/mobile-wallet-adapter-protocol-web3js/src/transact.ts

```diff
@@ -37,6 +44,27 @@ export interface Web3MobileWallet
         Web3SignTransactionsAPI,
         Web3SignMessagesAPI {}
 
+function getPayloadFromTransaction(transaction: LegacyTransaction | VersionedTransaction): Base64EncodedTransaction {
+    const serializedTransaction =
+        'version' in transaction
+            ? transaction.serialize()
+            : transaction.serialize({
+                  requireAllSignatures: false,
+                  verifySignatures: false,
+              });
+    const payload = fromUint8Array(serializedTransaction);
+    return payload;
+}
+
+function getTransactionFromWireMessage(byteArray: Uint8Array): Transaction | VersionedTransaction {
+    const version = VersionedMessage.deserializeMessageVersion(byteArray);
+    if (version === 'legacy') {
+        return Transaction.from(byteArray);
+    } else {
+        return VersionedTransaction.deserialize(byteArray);
+    }
+}
+
```

Comment:
@jstarry, does this look right to you? See below where these are used, to serialize/deserialize transactions to/from mobile wallets.